checker/tests/index/Pilot3ArrayCreation.java

```diff
@@ -1,7 +1,5 @@
 // This test case is for issue 44: https://github.com/kelloggm/checker-framework/issues/44
 
-// @skip-test until the test is fixed
-
 class Pilot3ArrayCreation {
     void test(int[] firstArray, int[] secondArray[]) {
         int[] newArray = new int[firstArray.length + secondArray.length];
```

Comment:
Would you please add more test cases?  Such as where one of the operands is positive and greater than negative one.